addons/pause/userscript.js

```diff
@@ -16,6 +16,14 @@ export default async function ({ addon, console, msg }) {
   onPauseChanged(setSrc);
 
   while (true) {
+    if (!document.querySelector(".pause-btn")) {
```

Comment:
Can we move this outside the while loop? I don't see why it needs to be here. This may create multiple event listeners in the same tab session.